pages/common/combine.md

```diff
@@ -0,0 +1,21 @@
+# combine
+
+> Perform set operations on lines of two files.
```

Comment:
```suggestion
> Compare lines of two files and print the result accordingly.
```

Reply:
To be perfectly honest, i like "Perform set operations on lines of two files." better :P This is because I believe that the command doesn't really do a comparison in a mathematical sense, but a set operation (ie. union, intersection, etc.)